### ISSUE TITLE
Learn Cypress testing framework for front-end testing

### ISSUE THREAD

**Issue by AlbinoDrake238** [EECE3093C/team-project-jam-frontend#30]
Cypress is the front end testing framework recommended by Vercel (the people/person who maintains next.js). Take 2-4 hours to familiarize yourself with the framework

**Comment by ItsAtlasX** [comment]
we could utilize chatGPT for unit test writing, saw in class that it can do fairly well with it.

**Comment by AlbinoDrake238** [comment]
Good idea Tyler, I'm not great with prompts for GPT but that's definitely something we can look into

Some resources I'm looking at:
Getting Started:
- https://saucelabs.com/resources/blog/getting-started-with-cypress
Examples:
- https://github.com/vercel/next.js/tree/canary/examples/with-cypress
